resources/serverless/templates/daemonset-delete-job.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.dockerRegistry.enableInternal }}
```

Comment:
Hm... it should be rather removed - it was only needed for 1.14.

Reply:
I'll remove it in separate PR.